openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/D19_ESSJun24/activity_eligibility/D19_ESSJun24_activity_eligibility_parameters.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from openfisca_core.variables import Variable
+from openfisca_core.periods import ETERNITY
+from openfisca_core.indexed_enums import Enum
+from openfisca_nsw_base.entities import Building
+
+
+
+class D19_ESSJun24_equipment_replaced(Variable):
+    value_type = bool
+    entity = Building
+    default_value = True
+    definition_period = ETERNITY
+    metadata = {
+      'display_question' : 'Is the activity the replacement of an existing gas water heater with an (air source) heat pump water heater?',
+      'sorting' : 1,
+      'eligibility_clause' : """A new installation is not eligible under ESS Activity D19. In ESS D19 the activity definition states that the activity must replace an existing gas water heater with an (air source) heat pump water heater. 
```

Comment:
Could you please clean this clause up: 
• it's missing the break at the end of the first line <br />  (refer to D17 for the use of the break)
• it's missing 'states' in the second sentence (Clause 1)